### ISSUE TITLE
Forwarding Yubikey

### ISSUE THREAD

**Issue by paweljasinski** [cgutman/USBIPServerForAndroid#5]
Hi,

I am doing work on forwarding a smartcard and yubikey based on your project.
I have taken a snapshot of the repo and converted it to gradle/android studio.
Until now I have found and hopefully fixed one bug which prevented attaching of yubikey.
For now limited communication with a key works, but it looks like there are more bugs which are exposed by yubikey.
I wonder what are your plans in respect to the project?
Are you interested in accepting PRs?

Cheers
--pawel

**Comment by cgutman** [comment]
Yep, I'm happy to accept PRs. I've done the conversion to Gradle and targetting a modern API level so PRs will not be too invasive to accept.
